### ISSUE TITLE
fetch_raw_logs() in api version 2

### ISSUE THREAD

**Issue by gunarka** [yozik04/vallox_websocket_api#21]
Hi, many thanks for this great api! I was using fetch_raw_logs() in api version 1.5.x without problems for several years.  
Now I would like to switch to api version 2 and tried to run the code example supplied in README.md.

However, I get the error message:

`Traceback (most recent call last):

  File ~/Programme/Vallox/unbenannt6.py:20 in <module>
    asyncio.get_event_loop().run_until_complete(run())

  File ~/Programme/anaconda3/lib/python3.9/asyncio/base_events.py:623 in run_until_complete
    self._check_running()

  File ~/Programme/anaconda3/lib/python3.9/asyncio/base_events.py:583 in _check_running
    raise RuntimeError('This event loop is already running')

RuntimeError: This event loop is already running`

**Comment by yozik04** [comment]
There is a mistake in your code somewhere. You should not do run two event loops simultaneously in one thread.

**Comment by gunarka** [comment]
Thanks for the quick reply!
Actually I just copied and executed the example code provided for
*fetch_raw_logs()
*on github:

import asynciofrom vallox_websocket_api import Client
client = Client('192.168.1.2')async def run():
    data = await client.fetch_raw_logs()

    from pprint import pprint
    pprint(data)asyncio.get_event_loop().run_until_complete(run())


...and got this error (as described before):
RuntimeError: This event loop is already running

I also got the same error when downlowding current metrics with*
fetch_metrics().*
However, I could get fetch_metrics() working by adding nest_asyncio to the
code.

import nest_asyncio
nest_asyncio.apply()

When adding nest_asyncio also to fetch_raw_logs() exmple above I got
another error, so it doesn't seem to be a good solution:

RuntimeError: cannot call recv while another coroutine is already waiting
for the next message

Am Sa., 17. Sept. 2022 um 09:18 Uhr schrieb Jevgeni Kiski <
***@***.***>:
